cobra/cmd/helpers.go

```diff
@@ -167,13 +185,20 @@ func guessProjectPath() {
 		// hardest case.. just a word.
 		if projectBase == "" {
 			x, err := getWd()
-			if err == nil {
-				projectPath = filepath.Join(x, inputPath)
-				return
+			if err != nil {
+				er(err)
 			}
-			er(err)
+			projectPath = filepath.Join(x, inputPath)
+			return
 		} else {
-			projectPath = filepath.Join(srcPath, projectBase, inputPath)
+			for _, srcPath := range srcPaths {
+				fpath := filepath.Join(srcPath, projectBase, inputPath)
+				if b, _ := exists(fpath); b {
+					projectPath = fpath
+					return
+				}
+			}
+			projectPath = filepath.Join(srcPaths[0], projectBase, inputPath)
```

Comment:
agreed, I was thinking `findProjectPath` but the existence of `guessProjectPath` makes it non-intuitive. 
just as an aside the existence of global variables `projectPath`, `inputPath` and `projectBase` make it harder to refactor things around. would mean a bigger change set but changing the structure around those variables would make things much cleaner imo. thoughts?